docs/api/sql/Raster-operators.md

```diff
@@ -1880,19 +1883,21 @@ Introduction: Returns the values at the given points or grid coordinates in the
 RS_Values is similar to RS_Value but operates on an array of points or grid coordinates.
 RS_Values can be significantly faster since a raster only has to be loaded once for several points.
 
+!!!Note
+    If the coordinate reference system (CRS) of the input `points` geometries differs from that of the `raster`, then `points` will be transformed to match the CRS of the `raster`.
+
 Format: 
 
 `RS_Values (raster: Raster, points: ARRAY[Geometry])`
 
 `RS_Values (raster: Raster, points: ARRAY[Geometry], band: Integer)`
 
-`RS_Values (raster: Raster, xCoordinates: ARRAY[Integer], yCoordinates: ARRAY[Integer], band: Integer)`
+```
+RS_Values (raster: Raster, xCoordinates: ARRAY[Integer], yCoordinates: ARRAY[Integer], band: Integer)
+```
 
 Since: `v1.4.0`
```

Comment:
@jiayuasu should I change the version number here?

Reply:
@furqaankhan No need to change.

Reply:
If I don't then it will update to the sedona.apache.org